src/bundle.js

```diff
@@ -49,22 +49,30 @@ module.exports = async (entryFile, dest, options = {}, log = () => {}) => {
 
   // set defaults, but allow override by passed in values
   const parcelBundleOptions = {
-    cache: true,
-    outDir: dest,
-    contentHash: true, // false if dev, true if prod ??
-    watch: false, // currently false if dev true if prod,
-    minify: false,
-    logLevel: 1,
+    entries: entryFile,
+    defaultConfig: require.resolve('@parcel/config-default'),
+    shouldDisableCache: false,
+    targets: {
+      action: {
```

Comment:
why `action` is it some parcel jargon or you choose the target key ?

Reply:
It can be anything.

Reply:
ok should we use web-assets key instead ?